BorrowToOwn.API/BorrowToOwn.Migrations/Data/20200821170522_added brand dbset for query purpose.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
i don't know Entity framework o, i sha see that u are renaming brand to brands.

Reply:
you are right boss ... created a brands table that maps to product , so it'll help the filter by brands fuctionality